src/vendor.ts

```diff
@@ -1,9 +1,7 @@
 // Polyfills
-import 'es6-promise';
-import 'es6-shim';
+import 'core-js';
 import 'reflect-metadata';
-import 'zone.js/dist/zone-microtask';
-import 'zone.js/dist/long-stack-trace-zone';
+import 'zone.js';
```

Comment:
I updated this to use the source version of these bundles in order to play well with webpack